All EIPIP Meetings/Meeting 065.md

```diff
@@ -0,0 +1,472 @@
+
+**Pooja Ranjan**
+* Okay, let me share the link to Panda, looks like he's around to have to join this meeting. We can, on and see when he joins and if he may have any thought to, we can get back to this one in the essence of time. Let's move on to item number 2. I'm assuming that we do not have any decision, or do we, on this topic. 
+
+**Golvin**
+* If I could back up for a second to the patent issue. I, I looked at the Creative Commons license again, and it, it specifically says that it does not, does not affect patent and trademark. and I think that's appropriate, that, that we just not have a policy. it's simply a specification. We don't endorse the specification, we don't certify. it's simply that, and it's totally on the user of the spec, to, to be aware of any patent or trademark issues. And it's perfectly fair for the author to hold patents, with the intent of, of, you know, licensing patents. That's, that's fair. It's a specification. That's all it is. 
+
+## What license to use for EIPs assets [29.57](https://youtu.be/0miajHm8JXs?t=1794)
+**Pooja Ranjan**
+* That sounds fair. And I believe the next topic that is listed as here for formal discussion is about license and yeah, what license do use for EIPs assets? We discussed this in the past, but I believe there were no, conclusion or decision on that, and we wanted to have a little bit more of discussion here. So, yeah. some, if you would like to give a little bit background on this topic, just jog the memory of everyone here. 
+
+**Sam**
+* Some editors want to force all assets to be cc0. And I, I don't want to do that. I want anything that isn't, copy left to be allowed for assets. 
+
+**Pooja Ranjan**
+* Greg, do you have any thoughts on this? 
+
+**Golvin**
+* I'm not a lawyer. 
+
+**Pooja Ranjan**
+* Again,. All right. I mean, on a high level, I personally think that when we are talking Ethereum and we are talking about having something as a standard and giving place Ethereum repository, we should try to make, things, as much as open source, free, no patent, no, like trademark things, so people can use it freely, Ethereum is an open source blockchain. We are like trying to, I mean, anyone is welcome to come and use it, so why to add things which may or may not have, permissions to be used or reused, actually. But that's my thought on that. I would be happy to hear other, editors thought on that. 
+
+**Sam**
+* Yeah, that, that, that's my thought as well. Like if somebody wants to come in with a, and and mandate that you have to use a certain algorithm in an EIPs and then, you know, a year later they start suing everybody, because they patented that algorithm, I think that's completely ridiculous and we should try to avoid that situation. 
```

Comment:
```suggestion
* Yeah, that's my thought as well. Like if somebody wants to come in with a mandate that you have to use a certain algorithm in an EIPs and then, you know, a year later they start suing everybody, because they patented that algorithm, I think that's completely ridiculous and we should try to avoid that situation. 
```